scripts/docs/generate_asset_documentation.py

```diff
@@ -333,6 +333,69 @@ def doc(self) -> snakemd.document.Document:
         return _doc
 
 
+class Categories:
+    """Categories structured by type"""
+    _categories = {}
+
+    def __init__(self):
+        self._categories[AssetType.ENVIRONMENT.value] = {}
+        self._categories[AssetType.COMPONENT.value] = {}
+        self._categories[AssetType.MODEL.value] = {}
+
+    def classify_asset(self, asset: AssetInfo):
+        for category in asset.categories:
+            cats = category.split("/")
```

Comment:
Can you give an example of a category value that includes a / ?

Reply:
I think this might be for the case where there are subcategories